.gitignore

```diff
@@ -9,5 +9,6 @@ target*
 .externalToolBuilders
 datasources/
 files.txz
+scripts.txz
```

Comment:
Why do you include scripts.txz ?

Reply:
because it was not included in the gitignore but is a side product of compilation. It must not be commited.